test/dpt_value_1_ucount_test.py

```diff
@@ -54,24 +54,3 @@ def test_from_knx_wrong_parameter2(self):
         """Test parsing of DPTValue1Ucount with wrong value (array containing string)."""
         with self.assertRaises(ConversionError):
             DPTValue1Ucount().from_knx(("0x23"))
```

Comment:
this is a duplicate of test/dpt_scene_number_test.py

Reply:
Might have been just a copy & paste error!